sospice/catalog/catalog.py

```diff
@@ -356,12 +356,24 @@ def plot_fov(self, ax, **kwargs):
             fovs.reset_index(inplace=True)
         else:
             fovs = Catalog(data_frame=self[list(required_columns)])
+        
+        fovs=fovs.sort_values(by=['DATE'])
+        Nfov=1*len(fovs)
+        if Nfov>12:
+            fovs=fovs[0:12]
```

Comment:
Could you run `make pre-commit`? This would standardize code formatting and do other kinds of validation